src/pages/Pool/index.tsx

```diff
@@ -62,15 +62,13 @@ const ButtonRow = styled(RowFixed)`
     width: 100%;
     flex-direction: row;
     justify-content: space-between;
-    flex-direction: row-reverse;
   `};
 `
 const PoolMenu = styled(Menu)`
   margin-left: 0;
   ${({ theme }) => theme.deprecated_mediaWidth.deprecated_upToSmall`
```

Comment:
it would be nice to move off the deprecated breakpoint while you're updating this, if you have time. 

we should at least be tracking this work in some linear tickets, i don't think we are right now D:

Reply:
Agreed! I've pushed a new commit which replaces the deprecated media queries and text components in this file. Will be good to incrementally update these usages as we work across the codebase